src/components/network-chart/index.js

```diff
@@ -3,38 +3,53 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // @flow
+import clamp from 'clamp';
+import { oneLine } from 'common-tags';
 import * as React from 'react';
+import {
+  TIMELINE_MARGIN_LEFT,
+  TIMELINE_MARGIN_RIGHT,
+} from '../../app-logic/constants';
 import explicitConnect from '../../utils/connect';
-import MarkerChartCanvas from './Canvas';
 import MarkerSettings from '../shared/MarkerSettings';
+import VirtualList from '../shared/VirtualList';
+import { withSize } from '../shared/WithSize';
 import NetworkChartEmptyReasons from './NetworkChartEmptyReasons';
+import NetworkChartRow from './NetworkChartRow';
+import memoize from 'memoize-immutable';
+import MixedTupleMap from 'mixedtuplemap';
 
 import {
   selectedThreadSelectors,
   getCommittedRange,
   getProfileInterval,
-  getPreviewSelection,
 } from '../../reducers/profile-view';
 import { getSelectedThreadIndex } from '../../reducers/url-state';
 import { updatePreviewSelection } from '../../actions/profile-view';
 
+import type { SizeProps } from '../shared/WithSize';
+import type { NetworkPayload } from '../../types/markers';
 import type {
   TracingMarker,
   MarkerTimingRows,
 } from '../../types/profile-derived';
 import type {
   Milliseconds,
   UnitIntervalOfProfileRange,
+  CssPixels,
 } from '../../types/units';
-import type { PreviewSelection } from '../../types/actions';
 import type {
   ExplicitConnectOptions,
   ConnectedProps,
 } from '../../utils/connect';
+import type { NetworkChartRowProps } from './NetworkChartRow';
 
 require('./index.css');
 
-const ROW_HEIGHT = 16;
+const ROW_HEIGHT = 17.5;
```

Comment:
In the CSS I point out a suggestion to use `px` rather than `em`. If you take that suggestion, perhaps it would be nice to have integer values for the height, instead of a 0.5.